### PR TITLE
Add support for xbacklight on Linux

### PR DESCRIPTION
**Pull request by bencevans**
Allows for changing the brightness without sudoing

Debian/Ubuntu users can install using `apt-get install xbacklight`

### CODE REVIEW

File: lib/linux-xbacklight.js [hunk 1]
```diff
@@ -0,0 +1,24 @@
+var xbacklight = require('xbacklight');
+var Promise = require('pinkie-promise');
+
+function get() {
+	return new Promise(function (resolve, reject) {
```
Comment (sindresorhus) [inline lib/linux-xbacklight.js]: There's no need to wrap a promise in another promise.

File: lib/linux-xbacklight.js [hunk 2]
```diff
@@ -0,0 +1,24 @@
+var xbacklight = require('xbacklight');
+var Promise = require('pinkie-promise');
```
Comment (sindresorhus) [inline lib/linux-xbacklight.js]: This is no longer needed as this module targets Node.js 4 now.

**Review by sindresorhus — CHANGES_REQUESTED** [review]

**Comment by sindresorhus** [comment]
What's the benefit of this over the existing Linux implementation?

**Comment by bencevans** [comment]
Doesn't require sudo rights

**Comment by sindresorhus** [comment]
But it does require a manual `apt-get install xbacklight`, which is also not very user-friendly. Any way you could bundle `xbacklight` in your module? I've done that in the past myself (https://github.com/sindresorhus/clipboardy/commit/e3404e7a27c5f9ec0d04d69c94e28faa0423715d), and seems to work fine.

**Comment by kevva** [comment]
ping @bencevans

**Comment by bencevans** [comment]
Shall try to take a look soon.

**Comment by bencevans** [comment]
I'm no longer using this so shall close the pull-request.

**Pull request closed without merging**